packages/storage/src/indexed-db/index.ts

```diff
@@ -223,46 +223,6 @@ export class IndexedDb implements IndexedDbInterface {
     }
   }
 
-  private addSctUpdates(txs: IbdUpdates, sctUpdates: ScanBlockResult['sctUpdates']): void {
```

Comment:
A lint in webstorm re-orders these fyi (alphabetically?)

Reply:
can we avoid lint/format changes that aren't defined in ci by eslint